task-2/src/components/Home/index.js

```diff
@@ -11,10 +13,14 @@ import './index.css';
 
 export const Home = () => {
 	const [openModal, setOpenModal] = useState(false);
-	const [selectedMovie, setSelectedMovie] = useState(undefined);
+
 	const [displayButton, setDisplayButton] = useState(false);
 	const show = () => setDisplayButton(true);
 
+	const selectedMovie = useSelector(
+		(state) => state.switchBannerReducer.selectedMovie,
```

Comment:
Consider creating a separate folder "selectors" under "src/store".
Then name and move all the arrow functions you pass to `useSelector` hooks there.
Think of them. the same way as you think of Action Creators:
Action Creators = functions that return Actions
Selectors = functions that (select and) return Parts of State

Reply:
- Selector file created.
- Selector has been separated. 
- Home component and movieDetailsBanner has been updated with the appropriate selectors.